scripts/prettyfy_json_files_in_dir.pl

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/perl -w
```

Comment:
Can we start to put this kind of file in a "utils" subfolder ?

Reply:
Good idea